plugin/src/events.ts

```diff
@@ -25,7 +25,7 @@ export function eventsApi(options: ShopifyPluginOptions) {
         const pageLinks: { url: string; rel: string }[] = paginationInfo
           .split(",")
           .map((pageData: string) => {
-            const [url, rel] = pageData.match(/<(.*)>; rel="(.*)"/) || ["", ""];
+            const [, url, rel] = pageData.match(/<(.*)>; rel="(.*)"/) || [];
```

Comment:
This was a bug lurking. Anyone with > 250 events since the last build would not have received all of them because this incorrect assignment resulted in a failure to correctly parse pagination data.